src/test/scala/uk/gov/nationalarchives/tdr/api/routes/ConsignmentRouteSpec.scala

```diff
@@ -390,6 +390,24 @@ class ConsignmentRouteSpec extends TestContainerUtils with Matchers with TestReq
       response.errors.head.message should equal(expectedResponse.errors.head.message)
   }
 
+  "getConsignment" should "return directories in the files list" in withContainers {
+    case container: PostgreSQLContainer =>
+      val utils = TestUtils(container.database)
+      val consignmentId = UUID.fromString("e72d94d5-ae79-4a05-bee9-86d9dea2bcc9")
+      utils.createConsignment(consignmentId, userId)
+      staticMetadataProperties.map(_.name).foreach(utils.addFileProperty)
+      clientSideProperties.foreach(utils.addFileProperty)
+      val topDirectory = UUID.fromString("ce0a51a5-a224-474f-b3a4-df75effd5b34")
+      val subDirectory = UUID.fromString("2753ceca-4df3-436b-8891-78ad38e2e8c5")
+      val fileId = UUID.fromString("6420152a-aaf2-4401-a309-f67ae35f5702")
+      createDirectoryAndMetadata(consignmentId, topDirectory, utils, "directory")
+      createDirectoryAndMetadata(consignmentId, subDirectory, utils, "subDirectory", topDirectory.some)
+      setUpFileAndStandardMetadata(consignmentId, fileId, utils, subDirectory.some)
+      val expectedResponse: GraphqlQueryData = expectedQueryResponse("data_files_and_folders")
```

Comment:
Maybe a line break here, just to split out the test setup from from the actual testing?